packages/atlas/src/providers/wallet/tmpwallet/consts.ts

```diff
@@ -0,0 +1,6 @@
+export const POLKADOT_CHAIN_ID = 'polkadot:91b171bb158e2d3848fa23a9f1c25182'
```

Comment:
removed this one, however I've left the default joystream chain Id just in case we'll have some issues with fetching the hash from the RPC (e.g bad connection etc)